### DISCUSSION TITLE
Is there a possibility for delayed initialization and startup of the LD2450?

### DISCUSSION THREAD

**Discussion by egoleos** [TillFleisch/ESPHome-HLK-LD2450#22] [Q&A]
Is it possible to delay the initialization and startup of the LD2450? 

Currently, the initialization of the LD2450 conflicts with the initialization of 4 I2C sensors. 

If I disable the LD2450, everything works fine, but when everything is together, the sensors are scanned but marked as FAILED. I suspect that the ESP32 is trying to handle everything at once, which it can't manage.

**Comment by TillFleisch** [comment]
No. The LD2450 immediately starts working after it has been powered. There is no way to tell it to suspend operation for long periods. The sensor does stop reporting values briefly if it is reset, maybe this can be used during boot as a workaround.

This could be related to a wiring issue. Make sure all components are wired correctly and they receive clean power. Changing the i2c frequency could also be beneficial.
